modules/new_profile/php/new_profile.class.inc

```diff
@@ -173,7 +173,7 @@ class New_Profile extends \NDB_Form
         $user_list_of_sites = $user->getData('CenterIDs');
         $num_sites          = count($user_list_of_sites);
         $psc_labelOptions   = array();
-        if ($num_sites > 1) {
+        if ($num_sites > 0) {
```

Comment:
I believe this is a wanted feature to ignore `DCC`.
Would be best to modify installation procedure to add a site. And also a page under Config menu to edit sites.

@driusan @ridz1208

Reply:
If this is a wanted feature to ignore DCC, then we should ignore DCC on the `Access Profile` page's drop down list, otherwise, the information is inconsistent.

Reply:
this is not designed to ignore the DCC this is made intentionally to hide the dropdown when the user is only affiliated to 1 site.

If user is affiliated to 1 site, that site is supposed to be selected by default and thus no need to display it on the front end. we agreed to undo that behaviour in an existing issue https://github.com/aces/Loris/issues/5380

The new behaviour should display the site list even if only 1 exists